rules.ini

```
@@ -150,6 +150,7 @@ EQU8 = (?:^|/)equ8_conf\.json$
 nProtect_GameGuard = (?:^|/)gameguard\.des$
 PunkBuster = (?:^|/)(?:PnkBstrA|pbsvc)\.exe$
 Ricochet = (?:^|/)Randgrid\.sys$
+Sentry = (?:^|/)sentry(?:/config\.toml|/sel\.dll|/splash\.png)
```

```suggestion
Sentry = (?:^|/)sentry(?:/config\.toml|/sel\.dll|/splash\.png|)
```
makes it also match when no extension are here